File: 백준/Bronze/1934. 최소공배수/최소공배수.c
#include <stdio.h>

int getM(int a, int b){
  int r, k = a * b;
  while(b != 0){
    r = a % b;
    a = b;
    b = r;
  }
  return k / a;
}

int main(void) {
  int a, b, c;
  scanf("%d", &a);

  for(int i = 0; i < a; i++){
    scanf("%d %d", &b, &c);
    printf("%d\n", getM(b, c));
  }

  return 0;
}

File: 백준/Bronze/1934. 최소공배수/최소공배수.py
def getM(a, b):
    k = a * b
    while b != 0:
        r = a % b
        a = b
        b = r
    
    return k // a

a = int(input())
for _ in range(a):
    b, c = map(int, input().split())
    print(getM(b, c))